stdlib/docs/crypto/hashes/blake3.md

```diff
@@ -1,6 +1,6 @@
-
+Initializes four memory addresses, provided for storing initial 4x4 blake3<br />state matrix ( i.e. 16 elements each of 32 -bit ), for computing blake3 2-to-1 hash<br /><br />Expected stack state:<br /><br />[state_0_3_addr, state_4_7_addr, state_8_11_addr, state_12_15_addr]<br /><br />Note, state_`i`_`j`_addr -> absolute address of {state[i], state[i+1], state[i+2], state[i+3]} in memory | j = i+3<br /><br />Final stack state:<br /><br />[...]<br /><br />Initialized stack state is written back to provided memory addresses.<br /><br />Functionally this routine is equivalent to https://github.com/itzmeanjan/blake3/blob/f07d32e/include/blake3.hpp#!L1709-L1713<br />
```

Comment:
I think there is still something off with how we process documentation. This comment belongs to an internal procedure and should not be printed out - but instead, it seems to be printed out as module-level comment.

Reply:
The only way that can happen (unless there is a bug, which I will check here) is if the doc comment for the local procedure starts on the first line of the module. The _only_ way a doc comment gets associated with the module and not a procedure is if it starts on the first line of the module and there is no separate doc comment for the procedure (or if someone manually sets the docs for the module). I had to do it this way because it is otherwise ambiguous whether the first doc comment in a file is associated with a module or the first documentable item in the module.

Rust distinguishes doc comments for modules vs module-scoped items with different prefixes, i.e. `//!` vs `///`. We may want to consider doing that if there are cases where the ambiguity causes a mixup like this.